core/domain/skill_domain_test.py

```diff
@@ -359,36 +368,6 @@ def test_misconception_id_range(self):
         self._assert_validation_error(
             'The misconception with id 5 is out of bounds')
 
-    def test_cannot_create_skill_change_class_with_invalid_change_list(self):
```

Comment:
Just to check, why are we deleting this and the following test functions?

Reply:
This is already being tested in `skill_domain_test.SkillChangeTests`